### DISCUSSION TITLE
windows 7-like window decorations?

### DISCUSSION THREAD

**Discussion by rosefloase** [Maplespe/DWMBlurGlass#348] [General]
hey, idiot that used to use windowblinds here
i can't really figure out how to make the default window decorations not look so... windows 10-like. they're still set as the default flat icons...
![image](https://github.com/Maplespe/DWMBlurGlass/assets/120997459/6cf829e9-918a-4c22-81af-0ee79d13e8ca)
i've seen multiple people on this discussion board showcase screenshots with aero style window decorations, and there's even a screenshot showcasing what i've been looking for in the official readme... under "Using third-party themes."
![image](https://github.com/Maplespe/DWMBlurGlass/assets/120997459/3f970b3b-8581-447c-8658-71f15018ce60)

can someone share how this is accomplished?

(edit): something to add, i'm on the 21H2 release of tiny10...
![image](https://github.com/Maplespe/DWMBlurGlass/assets/120997459/b365e747-eb99-4838-8467-71e20b6bedce)

**Comment by TheInsane101** [comment]
Install this theme (read all instructions in readme file): https://www.deviantart.com/vaporvance/art/Aero10-for-Windows-10-1903-22H2-909711949

Welcome to the beautiful world of msstyles theming! Say goodbye to Windowblinds forever!

**Reply by rosefloase** [reply]
thank you!!!!
![image](https://github.com/Maplespe/DWMBlurGlass/assets/120997459/b6d97644-6d5f-4770-8a38-a76793d85447)

**Reply by TheInsane101** [reply]
Glad I could help! Welcome to no-Windowblinds msstyles theming!

Turn off "use alternate navigation button style" in Oldnewexplorer for the windows 7 buttons. Consider also switching to the Aerexplorer alternative on Windhawk instead as well.
